data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/illuminates/illuminates_value_deep.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [illuminates]value=
+##
+# Actions:
+# Give all units 25% self-illuminates bonus damage to lawful units during the night
```

Comment:
"during the night" should be "deep underground"

Reply:
Updated.